include/dxc/Support/HLSLOptions.h

```diff
@@ -304,9 +305,9 @@ int ReadDxcOpts(const llvm::opt::OptTable *optionTable, unsigned flagsToInclude,
                 const MainArgs &argStrings, DxcOpts &opts,
                 llvm::raw_ostream &errors);
 
-/// Sets up the specified DxcDllSupport instance as per the given options.
-int SetupDxcDllSupport(const DxcOpts &opts, dxc::DxcDllSupport &dxcSupport,
-                       llvm::raw_ostream &errors);
+/// Sets up the specified DllLoader instance as per the given options.
+int SetupDllLoader(const DxcOpts &opts, dxc::SpecificDllLoader &dxcSupport,
+                   llvm::raw_ostream &errors);
```

Comment:
The comment doesn't match the function here - is the argument a `DllLoader` or a `SpecificDllLoader`? Also you've forward declared both of these above, but only one is used.